sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_file_system_client.py

```diff
@@ -258,13 +258,13 @@ def create_file_system(self, metadata=None,  # type: Optional[Dict[str, str]]
         :param public_access:
             To specify whether data in the file system may be accessed publicly and the level of access.
         :type public_access: ~azure.storage.filedatalake.PublicAccess
-        :keyword file_system_encryption_scope:
+        :keyword encryption_scope_options:
             Specifies the default encryption scope to set on the file system and use for
             all future writes.
 
             .. versionadded:: 12.9.0
 
-        :paramtype file_system_encryption_scope: dict or ~azure.storage.filedatalake.FileSystemEncryptionScope
+        :paramtype encryption_scope_options: dict or ~azure.storage.filedatalake.EncryptionScopeOptions
```

Comment:
@annatisch, I just wanted to mention you here to ensure your happy with this keyword name and the model name since there was a lot of discussion in the arch board review on it?

On its own, I think this name is fine, but I'm now also thinking about it in context of blob. In blob, this is called `ContainerEncryptionScope` which A) doesn't have `Options` and B) specifies `Container`. So, I almost prefer leaving in `FileSystem` and then maybe we wouldn't need `Options` since this one is now specific to a file system which means its more than a string? If we don't have `FileSystem`, we do need `Options` I think.

Maybe this is a bad thing to bring up now though since yesterday we seemed to decide all languages would call it `EncryptionScopeOptions` except Java which went with `FileSystemEncryptionScopeOptions` I think. :)

Reply:
I'm good with this new name - I think it's actually better than the blobs name, and I still feel the `file_system` part is implied by the nature of being in `create_file_system`...

Reply:
Sounds good. If you're happy with it, I'm happy with it :)